symplyphysics/laws/optics/interference_minimum.py

```diff
@@ -0,0 +1,36 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output, dimensionless)
+
+# Description
+## If the stroke difference is equal to an odd number of half-waves,
```

Comment:
What's `stroke`? Where can we see it?
What's `stroke difference`? Are they different in shape, size, color or what?